.idea/workspace.xml

```diff
@@ -3,10 +3,7 @@
   <component name="ChangeListManager">
     <list default="true" id="58b5dd2f-b754-499e-9d62-d2006c5185f8" name="Default Changelist" comment="">
       <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/workspace.xml" afterDir="false" />
-      <change beforePath="$PROJECT_DIR$/src/main/java/Tree/BinarySearchTree.java" beforeDir="false" afterPath="$PROJECT_DIR$/src/main/java/Tree/BinarySearchTree.java" afterDir="false" />
-      <change beforePath="$PROJECT_DIR$/src/main/java/Tree/Node.java" beforeDir="false" afterPath="$PROJECT_DIR$/src/main/java/Tree/Node.java" afterDir="false" />
-      <change beforePath="$PROJECT_DIR$/src/main/java/Tree/Tree.java" beforeDir="false" afterPath="$PROJECT_DIR$/src/main/java/Tree/Tree.java" afterDir="false" />
-      <change beforePath="$PROJECT_DIR$/src/test/java/Tree/BinarySearchTreeTest.java" beforeDir="false" afterPath="$PROJECT_DIR$/src/test/java/Tree/BinarySearchTreeTest.java" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/src/main/java/fizzbuzztree/fizzBuzzTree.java" beforeDir="false" afterPath="$PROJECT_DIR$/src/main/java/fizzbuzztree/fizzBuzzTree.java" afterDir="false" />

```

Comment:
You should REALLY run `git rm --cached .idea/workspace.xml` so that you stop adding/committing this file.